packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1137,9 +1137,13 @@ oie.select.authenticators.enroll.subtitle.custom = Security methods help protect
 
 oie.setup.required = Set up required
 oie.setup.optional = Set up optional
-oie.optional.authenticator.button.title = Finish
+oie.optional.authenticator.button.title = Set up later
 oie.verify.authenticator.button.text = Select
 oie.enroll.authenticator.button.text = Set up
+oie.enroll.authenticator.usage.text.access.recovery = Used for access and recovery
+oie.enroll.authenticator.usage.text.recovery = Used for recovery
```

Comment:
For enduser it may not be intuitive to understand what is `recovery` or `access` means while signing up or logging in.  @vishwasrikaanth-okta

Reply:
concern has been discussed offline and decided to leave the text as is.